### DISCUSSION TITLE
Референсы

### DISCUSSION THREAD

**Discussion by Darolga** [torin-asakura/drum-in#9] [General]
https://www.schoolofrock.com/music-lessons/drum-lessons
https://www.bimm.ac.uk/
https://www.bimm.ac.uk/drums/?tab=1
https://www.behance.net/gallery/124538031/Landing-page-redesign-for-the-drum-school?tracking_source=search_projects_recommended%7Cdrum%20school%20web%20design

https://www.behance.net/gallery/97717621/London-Music-School-New-Website?tracking_source=search_projects_recommended%7Cdrum%20school%20web%20design

https://www.behance.net/gallery/126601109/Music-School-landing-page?tracking_source=search_projects_recommended%7Cmusic%20school%20website

http://genesisexpo.wgl-demo.net/home-5/

Интересно подана биография  (горизонтальный скролл)
https://www.imreallyatrex.com/

Школы танцев, но стиль интересный
https://n.foxdsgn.com/pointe/
https://preview.themeforest.net/item/dropbeat-creative-dance-studio-wordpress-theme/full_screen_preview/33263146
https://preview.themeforest.net/item/dancy-dance-school-studio-elementor-template-kit/full_screen_preview/31930026

Стиль, цвет
https://www.behance.net/gallery/62768083/Design-Web-wynnr-2018

[Архив со скриншотами](https://drive.google.com/file/d/1MC15ccvNfX72YNbvGOJyrmIld8QAlDwV/view?usp=sharing)

**Comment by MarinaDrumin** [comment]
![image](https://user-images.githubusercontent.com/95244753/144438743-e814ff2d-a400-421d-89a4-18cc4d376d39.jpeg)

**Reply by MarinaDrumin** [reply]
Понравилось расположение и оформление частей

**Comment by MarinaDrumin** [comment]
![image](https://user-images.githubusercontent.com/95244753/144438901-f0e805c7-1cda-4238-8acd-0d9774e12213.png)

**Reply by MarinaDrumin** [reply]
Классный момент с закреплённым сверху меню

**Comment by MarinaDrumin** [comment]
![image](https://user-images.githubusercontent.com/95244753/144439057-b3ac53f5-646a-4f84-8f66-420f5d90fedb.jpeg)

**Reply by MarinaDrumin** [reply]
Разбивание фотки на триптих - очень круто!!!

**Reply by MarinaDrumin** [reply]
Нравится что шрифт мелкий- красиво смотрится, жаль что читается не оч

**Comment by MarinaDrumin** [comment]
![image](https://user-images.githubusercontent.com/95244753/144439278-13b80e4c-87dd-4042-868a-92e9940beaf2.jpeg)

**Reply by MarinaDrumin** [reply]
Нравится белый текст на фото и видео

**Comment by MarinaDrumin** [comment]
![image](https://user-images.githubusercontent.com/95244753/144439469-08419fda-bee1-4d17-881c-75cf289565cc.jpeg)

**Reply by MarinaDrumin** [reply]
Красивое сочетание цветов - фиолетовый и розовый. 
Думаю уйти от черно-красно-белого
Попробовать 
Фиолетовый- розовый- голубой например

**Comment by MarinaDrumin** [comment]
![image](https://user-images.githubusercontent.com/95244753/144439828-156bd657-c79d-4c1c-a718-88ca3389dee9.png)

**Reply by MarinaDrumin** [reply]
Хорошая цветовая гамма
Нравится чередование чёрного и белого фона

**Comment by MarinaDrumin** [comment]
Горизонтальный скрол в био тоже понравился!
